java/src/test/java/com/google/protobuf/LiteralByteStringTest.java

```diff
@@ -293,14 +291,21 @@ public void testNewOutput() throws IOException {
 
   public void testToString() throws UnsupportedEncodingException {
     String testString = "I love unicode \u1234\u5678 characters";
-    LiteralByteString unicode = new LiteralByteString(testString.getBytes(UTF_8));
-    String roundTripString = unicode.toString(UTF_8);
+    LiteralByteString unicode = new LiteralByteString(testString.getBytes(ByteString.UTF_8));
```

Comment:
This test seems exactly same as the testCharsetToString(). I suppose you meant to test unicode.toString("UTF-8") in this test case? The test for RopeByteString has the same problem.